packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -132,12 +136,27 @@ function SignInRoutes(): JSX.Element {
                 <LazySignUpContinue />
               </Route>
             </Route>
+            {SESSION_TASK_PATHS.map(path => (
```

Comment:
I need to be mindful of not including task components on the bundle by default, but only when the session transitions to have a task 

The complexity is on the fallback behavior, such as suspending on a shell once arriving on `/sign-in/add-organization`

Do we have any existing mechanisms similar to this case?